### ISSUE TITLE
[4.0] Monchrome template option seems pointless

### ISSUE THREAD

**Issue by ciar4n** [joomla/joomla-cms#26538]
I presume this option was added for accessibility however I can not find any evidence that there is an accessibility benefit to displaying everything in monochrome. All this option seems to do is simulate monochromatic vision rather than adding any aid.

**Comment by brianteeman** [comment]
There is a massive benefit.  People that are colorblind often find it difficult to distinguish between two different colors but easier to distinguish between two shades of gray.

Say a user sees both red and green as the same, they still may very well be able to differentiate between the specific shade of gray that represents red and the one that represents green.

**Comment by brianteeman** [comment]
It's also been part of ios since ios8 in the accessibility settings https://mcmw.abilitynet.org.uk/grayscale-iphoneipadipod-touch-ios-8 and macOS also in the accessibility settings.

In Windows 10 it can be found in the "ease of access" settings (apparently they thought that was a better term than accessibility.

**Comment by C-Lodder** [comment]
@brianteeman `People that are colorblind often find it difficult to distinguish between two different colors but easier to distinguish between two shades of gray`

Isn't this the reason why J4 uses WCAG 2.0 - AA Colour contrasts?

**Comment by C-Lodder** [comment]
Also, I haven't yet messed around with the colour settings on Atum, but in conjunction with https://github.com/joomla/joomla-cms/pull/26440 will the user be able to replicate a monochrome look and feel using the colour swatches?

**Comment by brianteeman** [comment]
Contrasts help to see the text on a button for example.
But if you cant see the difference between red and green then you cant see the different background color which is where the grayscale comes in

**Comment by C-Lodder** [comment]
Fair enough, in which case, is my second point not valid?

I think monochrome settings should be left down to the OS and monitor (if supported), not a site.

**Comment by brianteeman** [comment]
they dont need to use the color swatches as there is an option for the template which I dont see being removed in that pr
![image](https://user-images.githubusercontent.com/1296369/66575498-ebb54980-eb6d-11e9-9f1b-97ec26fe4005.png)

In addition it it ever gets accepted there is a pr for per user accessibility settings that also has a grayscale option

**Comment by C-Lodder** [comment]
@brianteeman What I'm saying is though, why would colour blind users perform this on a per-site basis when it can be controlled at the OS level globally?

Think of it as opening HTML files manually with your prefered browser each time when you can just set your prefered browser as the default.

**Comment by brianteeman** [comment]
Not all OS can do it

**Comment by C-Lodder** [comment]
If it's not native, there will be software, apps and/or monitor settings to do it.

**Comment by brianteeman** [comment]
Might as well not do anything then

**Comment by C-Lodder** [comment]
Well if I were colour blind, I'd rather sort my colour settings out at a global level. Waste of time having to find the setting for every website I use. Perhaps people do that

**Comment by ciar4n** [comment]
With the exception of iOS having a grayscale option placed under 'Accessibility', I can't find any objective evidence that greyscale is a benefit for those with color blindness. Android and Windows have options for grayscale but don't place them under 'Accessibility'.

**Comment by brianteeman** [comment]
Windows calls the accessibility settings "Ease of Access" and thats where the grayscale is

**Comment by ciar4n** [comment]
I'm gonna close this. As pointed out by @brianteeman, this appears to be a common a11y feature.  

I'd agree with @C-Lodder, that this feature is most likely switched on at an OS level if required so the need for such an option is still questionable. But that is probably a separate point to this issue.

**Comment by infograf768** [comment]
imho, setting such a feature at site level makes no sense.
The reason is simple: a site can be browsed or administered by colour blind and not colour blind people.
If the feature was a parameter set per user, as we do for preferred language or editor, it would make sense.

**Comment by brianteeman** [comment]
We have a per user setting in s yet to be merged pr

**Comment by infograf768** [comment]
PR # ?

**Comment by C-Lodder** [comment]
> If the feature was a parameter set per user, as we do for preferred language or editor, it would make sense.

There's a reason why big players like Microsoft, Google, Youtube, Twitter, Facebook, etc don't have monochrome settings on their websites. Same reason why they don't have their own screenreader. Because it's either controlled at OS level or people use software that applies this globally for everything they use.

**Comment by brianteeman** [comment]
There are more lines in this issue than in the code

**Comment by C-Lodder** [comment]
Tis what happens when you have pointless params. J3 seems to be calling
